examples/svgo-export/src/svgo-export.js

```diff
@@ -58,38 +61,26 @@ export function onExportSlices(context) {
 
 // ### Helper Functions
 
-// Utility function to run a command line command with a set of arguments.
-function runCommand(command, args) {
-  const task = NSTask.alloc().init()
-  task.launchPath = command
-  task.arguments = args
-  task.launch()
-  task.waitUntilExit()
-  return task.terminationStatus() == 0
-}
-
 // This is the function where we call out to svgo to do the heavy lifting (i.e: compress all SVG files in a given folder).
 //
 // svgo is installed installed in the Resources folder of our plugin so we need to get it from there.
 //
 // The SVGO options are based on our experience working with Sketch's exported SVGs, and to the best of our knowledge
 // they shouldn't effect the rendering of your assets, just reduce their size.
 function optimizeFolderWithSVGO(context, folderPath) {
-  const pathToSVGO = context.plugin.urlForResourceNamed(
-    'node_modules/.bin/svgo'
+  const pathToSVGO = context.plugin
+    .urlForResourceNamed('node_modules/svgo/bin/svgo')
+    .path()
+  return execSync(
+    `${pathToSVGO} --folder='${folderPath}' --pretty --disable=convertShapeToPath --enable=removeTitle --enable=removeDesc --enable=removeDoctype --enable=removeEmptyAttrs --enable=removeUnknownsAndDefaults --enable=removeUnusedNS --enable=removeEditorsNSData`
```

Comment:
btw I found compressing the whole folder a bit weird - I only want it to compress what I've exported as I might actually not want it to touch certain svg

Reply:
you are right. let's do that in another PR if it doesn't add complexity.

Reply:
sure, will patch